pkg/reconciler/ingress/resources/virtual_service.go

```diff
@@ -190,6 +190,8 @@ func makeVirtualServiceRoute(hosts sets.String, http *v1alpha1.HTTPIngressPath,
 		Route:   weights,
 		Timeout: types.DurationProto(http.Timeout.Duration),
 		Retries: &istiov1alpha3.HTTPRetry{
+			// TODO(https://github.com/knative/serving/issues/6367): Allow customization of this.
+			RetryOn:       strings.Join([]string{"5xx", "connect-failure", "refused-stream", "cancelled", "resource-exhausted", "retriable-status-codes"}, ","),
```

Comment:
nit: hoist to `var` and use that in tests.

nit: break the list onto separate lines for readability.

Reply:
Done